### PR TITLE
Rendering Techniques: Add instructions how to disable prop-types eslint rule

### PR DESCRIPTION
**Pull request by nikitarevenco**
## Because

The reader may find this useful if they want to disable prop-types eslint rule

## This PR

- Gives instructions on how to disable the eslint rule

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: react/getting_started_with_react/rendering_techniques.md [hunk 1]
```diff
@@ -102,12 +102,22 @@ function App() {
 ~~~
 
 <div class="lesson-note lesson-note--tip" markdown="1">
-  <h4>"Missing in props validation"</h4>
+  #### "Missing in props validation"
```
Comment (KevinMulhern) [inline react/getting_started_with_react/rendering_techniques.md:105]: Nit: the heading formatting is acting a little funky on the preview tool: 
Screenshot 2024-03-05 at 17 19 28


Outdenting should fix it:

```suggestion
#### "Missing in props validation"
```

**Review by KevinMulhern — COMMENTED** [review]
Solid change - thanks @nikitarevenco. Just a minor bit of formatting to fix and this is good to merge.

**Comment by nikitarevenco** [comment]
> Solid change - thanks @nikitarevenco. Just a minor bit of formatting to fix and this is good to merge.

Got it!

**Review by KevinMulhern — APPROVED** [review]
🚀

**Pull request merged**